dockers/docker-lldp/lldpmgrd

```diff
@@ -219,6 +242,18 @@ def main():
 
     lldpmgr.run()
 
+def resume_lldpd(self):
+    cmd = "lldpcli resume"
+    proc = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    (stdout, stderr) = proc.communicate()
+    if proc.returncode != 0:
+        self.log_error("Failed to resume lldpd with command: '{}': {}".format(cmd, stderr))
+        sys.exit(1)
```

Comment:
Is supervisord going to restart the process right away?

Reply:
No, this is the configuration on "/usr/share/sonic/templates/supervisord.conf.j2"

[program:lldpmgrd]
command=/usr/bin/lldpmgrd
priority=5
**autostart=false**
**autorestart=false**
stdout_logfile=syslog
stderr_logfile=syslog
dependent_startup=true
dependent_startup_wait_for=lldp-syncd:running